src/popup/components/AccountSwiperSlide.vue

```diff
@@ -12,15 +12,15 @@
       <BtnPlain
         v-if="idx !== 0"
         class="swiper-button prev"
-        @click="$emit('slide', idx - 1)"
+        @click.prevent="$emit('slide', idx - 1)"
```

Comment:
`.prevent` is used here so that parent button isn't triggered as well and the user doesn't get navigated to account-details